alchy/model.py

```diff
@@ -57,7 +52,14 @@ def __new__(mcs, name, bases, dct):
                     dct['__bind_key__'] = base['__bind_key__']
                     break
 
-        return DeclarativeMeta.__new__(mcs, name, bases, dct)
+        cls = DeclarativeMeta.__new__(mcs, name, bases, dct)
```

Comment:
Maybe add a comment here about why tablename is being set after call to `DeclarativeMeta.__new__`.